doc/topics/pillar/index.rst

```diff
@@ -168,6 +168,28 @@ And the actual pillar file at '/srv/pillar/common_pillar.sls':
     context.
 
 
+Dynamic Pillar Environments
+===========================
+
+If environment ``__env__`` is specified in :conf_master:`pillar_roots`, all
+environments that are not explicitly specified in :conf_master:`pillar_roots`
+will map to the directories from ``__env__``. This allows one to use dynamic
+git branch based environments for state/pillar files with the same file-based
+pillar applying to all environments. For example:
+
+.. code-block:: yaml
+
+    pillar_roots:
+      __env__:
+        - /srv/pillar
+
+    ext_pillar:
+      - git:
+        - __env__ https://example.com/git-pillar.git
+
+.. versionadded:: 2017.7.5
```

Comment:
This should be `.. versionadded:: 2017.7.5,2018.3.1` (no space after comma), since 2018.3.0 will only receive targeted bugfixes on top of 2018.3.0rc1.

Reply:
Done.